test/unit/specs/loginbutton.spec.js

```diff
@@ -11,11 +9,41 @@ describe('ti.facebook', () => {
 			button = Facebook.createLoginButton();
 		});
 
+		function checkValidAudiance(audience) {
```

Comment:
typo in the function name. checkValidAudiance -> checkValidAudience

Reply:
done